bundle/Command/MigrateCommand.php

```diff
@@ -0,0 +1,429 @@
+<?php
+/**
+ * NovaeZMailingBundle Bundle.
+ *
+ * @package   Novactive\Bundle\eZMailingBundle
+ *
+ * @author    Novactive <m.strukov@novactive.com>
+ * @copyright 2018 Novactive
+ * @license   https://github.com/Novactive/NovaeZMailingBundle/blob/master/LICENSE MIT Licence
+ */
+declare(strict_types=1);
+
+namespace Novactive\Bundle\eZMailingBundle\Command;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Novactive\Bundle\eZMailingBundle\Entity\Campaign;
+use Novactive\Bundle\eZMailingBundle\Entity\Mailing;
+use Novactive\Bundle\eZMailingBundle\Entity\MailingList;
+use Novactive\Bundle\eZMailingBundle\Entity\Registration;
+use Novactive\Bundle\eZMailingBundle\Entity\User;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Novactive\Bundle\eZMailingBundle\Core\IOService;
+use eZ\Publish\Core\Repository\Repository;
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+
+/**
+ * Class MigrateCommand.
+ */
+class MigrateCommand extends Command
+{
+    /**
+     * @var IOService
+     */
+    private $ioService;
+
+    /**
+     * @var SymfonyStyle
+     */
+    private $io;
+
+    /**
+     * @var EntityManagerInterface
+     */
+    private $entityManager;
+
+    /**
+     * @var Repository;
+     */
+    private $ezRepository;
+
+    /**
+     * @var ConfigResolverInterface
+     */
+    private $configResolver;
+
+    public const CAMPAIGN_LIST_CONTENT_ID = 53;
```

Comment:
this must me remove right? can we automatically fallback on content ID 1 if somehow the ID is not found?

Reply:
Changed to Content Id = 1